modules/post/windows/manage/shellcode_inject.rb

```diff
@@ -26,18 +25,17 @@ def initialize(info={})
       [
         OptPath.new('SHELLCODE', [true, 'Path to the shellcode to execute']),
         OptInt.new('PID', [false, 'Process Identifier to inject of process to inject the shellcode. (0 = new process)', 0]),
-        OptBool.new('CHANNELIZED', [true, 'Retrieve output of the process', true]),
-        OptBool.new('INTERACTIVE', [true, 'Interact with the process', true]),
+        OptBool.new('CHANNELIZED', [true, 'Retrieve output of the process', false]),
+        OptBool.new('INTERACTIVE', [true, 'Interact with the process', false]),
         OptBool.new('HIDDEN', [true, 'Spawn an hidden process', true]),
-        OptBool.new('AUTOUNHOOK', [true, 'Auto remove EDRs hooks', true]),
+        OptBool.new('AUTOUNHOOK', [true, 'Auto remove EDRs hooks', false]),
```

Comment:
FYI the unhook dll is currently not working.. see https://github.com/rapid7/metasploit-payloads/pull/366 for a potential fix.

do you maybe know the reason of that particular `if`?